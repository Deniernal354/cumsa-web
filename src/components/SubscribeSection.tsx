"use client";
import NewsletterEmbed from "./NewsletterEmbed";
import { FORMS } from "@/lib/constants";

export default function SubscribeSection() {
  return (
    <div className="container-base py-16">
      <NewsletterEmbed title="Subscribe to the MSA Newsletter" formUrl={FORMS.newsletter} />
      <NewsletterEmbed
        title="Subscribe to the MSA Alumni/Community Newsletter"
        formUrl={FORMS.alumniNewsletter}
      />
    </div>
  );
}
