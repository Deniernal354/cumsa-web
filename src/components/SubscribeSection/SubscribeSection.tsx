import NewsletterEmbed from "../NewsletterEmbed/NewsletterEmbed";

export default function SubscribeSection() {
  return (
    <section className="py-16 bg-[var(--surface)]">
      <div className="container-base max-w-2xl">
        <h2 className="text-2xl font-bold mb-4 text-center">Subscribe to our Newsletter</h2>
        <NewsletterEmbed />
      </div>
    </section>
  );
}
