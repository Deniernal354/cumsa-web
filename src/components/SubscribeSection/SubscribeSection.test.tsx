import { render } from "@testing-library/react";
import SubscribeSection from "./SubscribeSection";

test("SubscribeSection snapshot", () => {
  const { asFragment } = render(<SubscribeSection />);
  expect(asFragment()).toMatchSnapshot();
});
