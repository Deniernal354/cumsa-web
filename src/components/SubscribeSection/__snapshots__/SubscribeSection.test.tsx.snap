// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubscribeSection snapshot 1`] = `
<DocumentFragment>
  <section
    class="py-16 bg-[var(--surface)]"
  >
    <div
      class="container-base max-w-2xl"
    >
      <h2
        class="text-2xl font-bold mb-4 text-center"
      >
        Subscribe to our Newsletter
      </h2>
      <div
        class="rounded-xl overflow-hidden shadow ring-1 ring-black/10 bg-white p-4"
      >
        <iframe
          allowfullscreen=""
          loading="lazy"
          src="https://embeds.beehiiv.com/1b1e2e2b-2e2e-4e2e-8e2e-2e2e2e2e2e2e?slim=true"
          style="width: 100%; min-height: 320px; background: transparent;"
          title="Newsletter Signup"
        />
      </div>
    </div>
  </section>
</DocumentFragment>
`;
